dynaflow/src/main/java/com/powsybl/dynaflow/DynaFlowConfig.java

```diff
@@ -36,10 +37,13 @@ public static DynaFlowConfig fromPropertyFile() {
 
     public static DynaFlowConfig fromPlatformConfig(PlatformConfig platformConfig) {
         Objects.requireNonNull(platformConfig);
-        ModuleConfig config = platformConfig.getModuleConfig("dynaflow");
-        Path homeDir = config.getPathProperty("homeDir");
-        boolean debug = config.getBooleanProperty("debug", DEFAULT_DEBUG);
-        return new DynaFlowConfig(homeDir, debug);
+        Optional<ModuleConfig> optionalModuleConfig = platformConfig.getOptionalModuleConfig("dynaflow");
+        if (optionalModuleConfig.isPresent()) {
+            Path homeDir = optionalModuleConfig.get().getPathProperty("homeDir");
+            boolean debug = optionalModuleConfig.get().getBooleanProperty("debug", DEFAULT_DEBUG);
+            return new DynaFlowConfig(homeDir, debug);
+        }
+        return null;
```

Comment:
You can use `Optional::map` here to have something more compact:
```suggestion
        return platformConfig.getOptionalModuleConfig("dynaflow")
                .map(config -> new DynaFlowConfig(config.getPathProperty("homeDir"), config.getBooleanProperty("debug", DEFAULT_DEBUG)))
                .orElse(null);
```